src/internal.c

```diff
@@ -1073,7 +1073,7 @@ static INLINE int CalcRsaInverses(RsaKey* key)
     return ret;
 }
 
-
+#ifndef WOLFSSH_NO_RSA
```

Comment:
Move this up to include the CalcRsaInverses function.

Reply:
done